auth-jwt-service/src/main/java/io/stargate/auth/jwt/AuthzJwtService.java

```diff
@@ -117,7 +117,8 @@ public void authorizeDataWrite(String token, String keyspace, String table, Scop
 
   /** {@inheritdoc} */
   @Override
-  public void authorizeDataWrite(String token, List<TypedKeyValue> typedKeyValues, Scope scope)
+  public void authorizeDataWrite(
+      String token, String keyspace, String table, List<TypedKeyValue> typedKeyValues, Scope scope)
```

Comment:
I'm a bit lost... the newly added `keyspace` and `table` parameters do not seem to be used... did I miss something?

Reply:
For a JWT we're assuming that a role+permissions have been setup meaning that the keyspace and table are not used. But for extensibility to add a more advanced auth provided like say, OPA, you'll need to know the keyspace and table that are being accessed.